packages/frontend/src/pages/Settings.tsx

```diff
@@ -189,124 +189,99 @@ const Settings: FC = () => {
-                                            }
-                                            to="/generalSettings/userManagement"
-                                            exact
-                                            icon={
-                                                <MantineIcon
-                                                    icon={IconUserPlus}
-                                                />
-                                            }
-                                        />
-                                    )}
-                                    {user.ability.can(
-                                        'manage',
-                                        subject('Organization', {
-                                            organizationUuid:
-                                                organization.organizationUuid,
-                                        }),
-                                    ) && (
-                                        <RouterNavLink
-                                            label={
-                                                isGroupManagementEnabled
-                                                    ? 'User & group attributes'
-                                                    : 'User attributes'
-                                            }
-                                            to="/generalSettings/userAttributes"
-                                            exact
-                                            icon={
-                                                <MantineIcon
-                                                    icon={IconUserShield}
-                                                />
-                                            }
-                                        />
-                                    )}
+                                {user.ability.can(
+                                    'update',
+                                    'OrganizationMemberProfile',
+                                ) && (
+                                    <RouterNavLink
+                                        label={
+                                            isGroupManagementEnabled
+                                                ? 'Users & groups'
+                                                : 'User management'
+                                        }
+                                        to="/generalSettings/userManagement"
+                                        exact
+                                        icon={
+                                            <MantineIcon icon={IconUserPlus} />
+                                        }
+                                    />
+                                )}
+                                {user.ability.can(
+                                    'manage',
+                                    subject('Organization', {
+                                        organizationUuid:
```

Comment:
reundant check, this should be 
`organizationUuid: user.data?.organizationUuid` I think.

Reply:
I didn’t touch this chunk (the indentation diff might make it seem otherwise), but I believe it's correct because this is the permission definition in CASL:

```ts
can('manage', 'Organization', {
    organizationUuid: member.organizationUuid,
});
```

To match that, it should be `organization.organizationUuid`

or am I wrong?